qiskit_experiments/test/fake_backend.py

```diff
@@ -55,12 +55,25 @@ def target(self) -> Target:
         return self._target
 
     def run(self, run_input, **options):
+        if not isinstance(run_input, list):
```

Comment:
I think we always submit circuits as lists even in the case of a single circuit, but this matches the behavior of `IBMBackend` so it's valid.

Reply:
That sounds right. Because the name was `run_input` instead of `circuits`, I wasn't sure what was possible, so I checked the other `Backend` classes and decided to handle `QuantumCircuit` and `list[QuantumCircuit]`.